README.md

```diff
@@ -1,22 +1,82 @@
 # webknossos-connect
-A [webKnossos](https://github.com/scalableminds/webknossos) compatible data connector written in Python
+A [webKnossos](https://github.com/scalableminds/webknossos) compatible data connector written in Python. 
+
+webKnossos-connect serves as an adapter between the webKnossos data store interface and other alternative data storage servers (e.g BossDB) or static files hosted on Google Cloud Storage (e.g. Neurodata Precomputed)
 
 [![CircleCI](https://circleci.com/gh/scalableminds/webknossos-connect.svg?style=svg&circle-token=1d7b55b40a5733c7563033064cee0ed0beef36b6)](https://circleci.com/gh/scalableminds/webknossos-connect)
 
+Available Adapaters / Supported Data Formats:
+- [BossDB](https://bossdb.org/)
+- [Neuroglancer Precomputed](https://github.com/google/neuroglancer/tree/master/src/neuroglancer/datasource/precomputed)
+
 ## Usage
+### 1. Installation / Docker
+Install webKnossos-connect using Docker or use the instructions for native installation below.
+`docker-compose up --build webknossos-connect`
 
-1. Add webknossos-connect to the webKnossos database:
+### 2. Connecting to webKnossos
+Register your webknossos-connect instance with your main webKnossos instance. Modify the webKnossos Postgres database:
   ```
   INSERT INTO "webknossos"."datastores"("name","url","key","isscratch","isdeleted","isforeign","isconnector")
-  VALUES (E'connect',E'http://localhost:8000',E'secret-key',FALSE,FALSE,FALSE,TRUE);
+  VALUES (E'connect', E'http://localhost:8000', E'secret-key', FALSE, FALSE, FALSE, TRUE);
   ```
-2. To use the initial datasets from [neurodata.io](https://neurodata.io/ndcloud/#data), create a `.env` file:
+### 3. Adding Datasets
+Add and configure datasets to webKnossos-connect to make them available for viewing in webKnossos
+
+#### 3.1 REST API
```

Comment:
I would propose to change the order of the points:
1. default datasets
2. webknossos UI
3. (if at all) API endpoint. I would actually not publicly recommend to use it. Imo it should be an implementation detail and accessed through the wk UI.